### DISCUSSION TITLE
A minimum and maximum for hasMany in relationship

### DISCUSSION THREAD

**Discussion by Chrinsieboy** [payloadcms/payload#1938] [Feature Requests & Ideas]
When you have a relationship to staffmembers and you want a minimum then you cant do that at the moment.
Maybe add a feature so you can like this: 

```typescript
{
  slug: 'staff-collection',
  fields: [
    {
      name: 'staff',
      type: 'relationship',
      relationTo: 'members',
      hasMany: true,
      admin: {
          // option 1
          minItems: 1,
          maxItems: 3,

          // option 2
          min: 1,
          max: 3
      }
    }
  ]
}
```

**Comment by DanRibbens** [comment]
I needed this so I made it a reality: https://github.com/payloadcms/payload/pull/2237

Since validation happens in both the admin UI and the backend I opted to put`min` and `max` at the top instead of within  `admin`.
